estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,73 @@
+from odoo import fields, models, api
+from odoo.exceptions import UserError
+from datetime import timedelta
+
+
+class EstatePropertyOfferModel(models.Model):
+    _name = "estate.property.offer"
+    _description = "The estate property offers model"
+    _order = "price desc"
+
+    price = fields.Float(string="Offer Price", required=True)
+    status = fields.Selection(
+        [("accepted", "Accepted"), ("refused", "Refused")], string="Status", copy=False
+    )
+    partner_id = fields.Many2one("res.partner", string="Buyer", required=True)
+    property_id = fields.Many2one("estate.property", string="Property", required=True)
+    property_type_id = fields.Many2one(
+        related="property_id.property_type_id", store=True
```

Comment:
```suggestion
        related="property_id.property_type_id",
```
store=True in Many2one field? I don't think so..